src/main/java/org/zalando/fahrschein/domain/Cursor.java

```diff
@@ -1,6 +1,6 @@
 package org.zalando.fahrschein.domain;
 
-public final class Cursor {
+public class Cursor {
```

Comment:
Why no longer `final`? There is no need to mock this class since you can just as easily create an instance.